controllers/configuration_controller_test.go

```diff
@@ -764,6 +764,44 @@ func TestAssembleTerraformJob(t *testing.T) {
 	assert.Equal(t, containers[1].Image, "d")
 }
 
+func TestTfStatePropertyToToProperty(t *testing.T) {
+	testcases := []TfStateProperty{
+		{
+			Value: 123,
+			Type:  "b",
+		},
+		{
+			Value: 123.1,
+			Type:  "b",
+		},
+		{
+			Value: "123",
+			Type:  "b",
+		},
+		{
+			Value: true,
+			Type:  "b",
+		},
+		{
+			Value: []interface{}{"21", "aaa", 12, true},
+			Type:  "b",
+		},
+	}
```

Comment:
Can you add a map value to the testcases?

Reply:
ok